src/inference/dev_api/openvino/runtime/iplugin.hpp

```diff
@@ -212,6 +212,12 @@ class OPENVINO_RUNTIME_API IPlugin : public std::enable_shared_from_this<IPlugin
      */
     const std::shared_ptr<ov::threading::ExecutorManager>& get_executor_manager() const;
 
+    /**
+     * @brief Check if plugin support mmap for cached model reading. Returns false is the method is not overrided by plugin.
+     * @return true if mmap is supported, false otherwise
+     */
+    virtual bool support_mmap_for_caching() const;
```

Comment:
Use property instead adding functions?
`util::contains(plugin.get_property(ov::supported_properties), ov::enable_mmap.name())` to check if supprted and if yes then get it value.

And add ENABLE_MMAP to supported properties in GPU plugin

Reply:
I would not like to show this "property" via public API. 
It just says whether mmap should be used for a particular plugin.